public/javascripts/labelingGuidePanelResize.js

```diff
@@ -0,0 +1,49 @@
+function checkWindowSize(){
+    var w = document.documentElement.clientWidth;
+    var panel = document.getElementById("help-panel");
+    if (w < 978) {
```

Comment:
Where are the numbers (978, 1184, 250, 275, 500, etc) coming from? Are they mostly just trial and error?

Are any derived from the width/height of HTML elements? If so, could you try to get the number from the HTML element itself instead of a hard coded number?

Do any of the numbers have some sort of physical meaning that could be described (maybe the max width/height we would allow something to be)? If so, could you put the hard coded numbers into variables with descriptive names?